oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -477,7 +523,7 @@ static RunningContext instance() {
                 .build();
         }
 
-        MeterEntity meterEntity;
+        private Map<MeterEntity, Sample[]> meterSamples;
```

Comment:
This could be never initialized, if there is no `service`/`instance`/`endpoint` called. Then could cause NPE in the Analyzer.

Reply:
done